### ISSUE TITLE
ST4 builtin syntax provides same functionality

### ISSUE THREAD

**Issue by deathaxe** [aurule/json-kv#4]
ST4's JSON syntax 

1. scopes keys `meta.mapping.key` and values `meta.mapping.value`. With some tweaks of color schemes keys and values can easily be distinguished.
2. is implemented based on official specifications to provide proper scopes.

**Comment by Monox18** [comment]
Yes, that's right. ST4 has updated and implemented a new syntax making it a conflict with those packages that created a new syntax. Affecting #5 and #2 .

That's why I created [Json Symbols and Color Syntax](https://github.com/Monox18/JsonSymbolsAndColorSyntax). It extends the oficial syntax.

I have already pushed it to Sublime's Package Control and will probably take weeks before they merge it. In the meantime, you can try out by simply downloading it to your Packages folder.
